client/me/help/gm-closure-notice/index.jsx

```diff
@@ -89,7 +89,7 @@ const GMClosureNotice = ( {
 	const MAIN_MESSAGES = {
 		before: {
 			hasPlan: translate(
-				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. Email support will remain open.',
+				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. customer support via email will remain open.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.